fv3core/utils/gt4py_utils.py

```diff
@@ -207,7 +207,7 @@ def make_storage_from_shape(
     *,
     dtype: DTypes = np.float64,
     init: bool = True,
-    mask: Tuple[bool, bool, bool] = (True, True, True),
+    mask: Optional[Tuple[bool, bool, bool]] = None,
```

Comment:
Is this because some are 2d fields now? Or rather, what is this change about?

Reply:
This is to force `mask` to be computed from the shape if it is not specifically supplied, with the goal of minimizing changes to existing code.